hikari/impl/entity_factory.py

```diff
@@ -2618,6 +2682,13 @@ def deserialize_autocomplete_interaction(
             member = None
             user = self.deserialize_user(payload["user"])
 
+        authorizing_integration_owners = {
+            application_models.ApplicationIntegrationType(int(integration_type)): snowflakes.Snowflake(
+                int(integration_owner_id)
+            )
```

Comment:
I believe for snowflakes, you do not need to cast it to an int before hand (as Snowflake is a subclass of int)